src/EventListener/Doctrine/LineItemLoaderListener.php

```diff
@@ -43,7 +43,7 @@ public function postLoad(Assignment $assignment): void
     {
         //we're forcing the doctrine to load the line item by using cache instead of database query
         $assignment->setLineItem(
-            $this->lineItemRepository->findOneById($assignment->getLineItemId())
+            $this->lineItemRepository->findOneById($assignment->getLineItem()->getId())
```

Comment:
I think that we have a cache issue here. This call will make a DB query I think. Can you check it, please?

Reply:
I am going to profile it, good catch!